source/OctoPack.Tasks/GetAssemblyVersionInfo.cs

```diff
@@ -128,18 +126,6 @@ private TaskItem UseAssemblyVersion(FileVersionInfo info)
                 {"Version", assemblyVersionInfo},
             });
         }
-
-
-        private static string GetNuGetVersionFromGitVersionInformation(string path)
```

Comment:
Moved to [AssemblyExtentions.cs](https://github.com/OctopusDeploy/OctoPack/pull/99/files#diff-7b14147167ee9cfb17070e0cff2f7842).